extra/3-magic-8-ball.js

```diff
@@ -58,10 +62,50 @@ function shakeBall() {
 
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
-function checkAnswer(answer) {
+
+const answer = [
```

Comment:
My advice, break answers array into four array and then join them into one using spread operator.